samples/boards/nrf/nrfx/src/main.c

```diff
@@ -77,35 +82,39 @@ void main(void)
 
 	LOG_INF("nrfx_gpiote initialized");
 
-	/* Initialize DPPI channel */
+	/* Allocate a (D)PPI channel. */
+#if defined(DPPI_PRESENT)
 	uint8_t channel;
-
 	err = nrfx_dppi_channel_alloc(&channel);
+#else
+	nrf_ppi_channel_t channel;
+	err = nrfx_ppi_channel_alloc(&channel);
+#endif
 	if (err != NRFX_SUCCESS) {
-		LOG_ERR("nrfx_dppi_channel_alloc error: %08x", err);
+		LOG_ERR("(D)PPI channel allocation error: %08x", err);
 		return;
 	}
 
-	/* Configure input pin to publish to the DPPI channel and output pin to
-	 * receive events from the DPPI channel. Note that output pin is
-	 * subscribed using the OUT task. This means that each time the button
-	 * is pressed, the LED pin will be toggled.
+	/* Configure endpoints of the channel so that the input pin event is
```

Comment:
I had the impression that "endpoints" (event and task) were a terminology that made sense only for the PPI peripheral; DPPI, instead, would have the publish and subscribe terminology. But it seems that now, nrfx has this _gppi_ infix for common APIs and endpoint is a generic terminology...